TwentyTwentyTwenty/Program.cs

```diff
@@ -35,6 +38,9 @@ public MyCustomApplicationContext()
                 Visible = true
             };
 
+            var soundPlayer = File.Exists(SoundPath)
```

Comment:
We have a race condition here since, if I understand SoundPlayer correctly, the file is not loaded until played the first time which is about 20 minutes after startup. How about doing soundplayer.Load(...) here and handling filenotfound gracefully?

Reply:
.... or even LoadAsync could be preferrable.

Reply:
No, LoadAsync [won't work](https://docs.microsoft.com/en-us/dotnet/api/system.media.soundplayer.loadasync?view=net-5.0#remarks): "If a SoundPlayer is configured to load a .wav file from a local file path, this method does nothing, because loading is deferred until playing begins."

Reply:
Good point. Just pushed an update, handling the known exceptions and appending to the tray icon text.
"Should" never happen, but nice to know if it does